browser/about_flags.cc

```diff
@@ -0,0 +1,375 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/about_flags.h"
+
+#include "base/strings/string_util.h"
+#include "brave/browser/brave_features_internal_names.h"
+#include "brave/common/brave_features.h"
+#include "brave/components/brave_ads/common/features.h"
+#include "brave/components/brave_component_updater/browser/features.h"
+#include "brave/components/brave_rewards/common/buildflags/buildflags.h"
+#include "brave/components/brave_rewards/common/features.h"
+#include "brave/components/brave_shields/common/features.h"
+#include "brave/components/brave_sync/buildflags/buildflags.h"
+#include "brave/components/brave_talk/features.h"
+#include "brave/components/brave_vpn/buildflags/buildflags.h"
+#include "brave/components/brave_wallet/common/buildflags/buildflags.h"
+#include "brave/components/decentralized_dns/buildflags/buildflags.h"
+#include "brave/components/ipfs/buildflags/buildflags.h"
+#include "brave/components/ntp_background_images/browser/features.h"
+#include "brave/components/sidebar/buildflags/buildflags.h"
+#include "brave/components/speedreader/buildflags.h"
+#include "net/base/features.h"
+
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
+#include "brave/components/brave_vpn/features.h"
+#endif
+
+#if BUILDFLAG(ENABLE_SIDEBAR)
+#include "brave/components/sidebar/features.h"
+#endif
+
+#if BUILDFLAG(ENABLE_SPEEDREADER)
+#include "brave/components/speedreader/features.h"
+#endif
+
+#if BUILDFLAG(ENABLE_BRAVE_SYNC)
+#include "brave/components/brave_sync/features.h"
+#endif
+
+#if BUILDFLAG(ENABLE_IPFS)
+#include "brave/components/ipfs/features.h"
+#endif
+
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
+#include "brave/components/brave_wallet/common/features.h"
+#endif
+
+#if BUILDFLAG(DECENTRALIZED_DNS_ENABLED)
+#include "brave/components/decentralized_dns/features.h"
+#endif
+
+using brave_shields::features::kBraveAdblockCnameUncloaking;
+using brave_shields::features::kBraveAdblockCollapseBlockedElements;
+using brave_shields::features::kBraveAdblockCosmeticFiltering;
+using brave_shields::features::kBraveAdblockCosmeticFilteringNative;
+using brave_shields::features::kBraveAdblockCspRules;
+using brave_shields::features::kBraveDomainBlock;
+using brave_shields::features::kBraveExtensionNetworkBlocking;
+using ntp_background_images::features::kBraveNTPBrandedWallpaper;
+using ntp_background_images::features::kBraveNTPBrandedWallpaperDemo;
+using ntp_background_images::features::kBraveNTPSuperReferralWallpaper;
+
+namespace flag_descriptions {
+
```

Comment:
should be also in anonymous namespace here, because these strings are not used anywhere outside this file (and shouldn't be visible to the linker).

Reply:
Oops! True that, fixed!